baseframe/static/js/baseframe-material.js

```diff
@@ -1,15 +1,11 @@
 // This is a global function. Isn't there a better way to do this?
 
 function activate_widgets() {
-  // Activate select2.js for non-mobile browsers
-  if (!Modernizr.touch) {
-    $('select:not(.notselect)').select2();
 
-    //Remove the scroll lock class added by mui.js on opening select menu
-    $('select:not(.notselect)').on("select2:select", function(e) {
-      $('body').removeClass('mui-scroll-lock');
-    });
-  }
+  //Remove the scroll lock class added by mui.js on opening select dropdown
+  $('select').on("select2:select", function(e) {
```

Comment:
@vidya-ram can you confirm this change does not impact apps that use Select2?

Reply:
Yes, it works fine in Lastuser and HGTV